x-pack/test/functional/apps/discover/saved_searches.ts

```diff
@@ -16,28 +16,41 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const dataGrid = getService('dataGrid');
   const panelActions = getService('dashboardPanelActions');
   const panelActionsTimeRange = getService('dashboardPanelTimeRange');
+  const queryBar = getService('queryBar');
+  const filterBar = getService('filterBar');
   const ecommerceSOPath = 'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json';
+  const defaultSettings = {
+    defaultIndex: 'logstash-*',
+    'doc_table:legacy': false,
+  };
 
-  // FLAKY https://github.com/elastic/kibana/issues/104578
-  describe.skip('Discover Saved Searches', () => {
+  const setTimeRange = async () => {
+    const fromTime = 'Apr 27, 2019 @ 23:56:51.374';
+    const toTime = 'Aug 23, 2019 @ 16:18:51.821';
+    await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+  };
+
+  describe('Discover Saved Searches', () => {
     before('initialize tests', async () => {
       await esArchiver.load('x-pack/test/functional/es_archives/reporting/ecommerce');
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
       await kibanaServer.importExport.load(ecommerceSOPath);
-      await kibanaServer.uiSettings.update({ 'doc_table:legacy': false });
+      await kibanaServer.uiSettings.replace(defaultSettings);
     });
+
     after('clean up archives', async () => {
       await esArchiver.unload('x-pack/test/functional/es_archives/reporting/ecommerce');
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
       await kibanaServer.importExport.unload(ecommerceSOPath);
       await kibanaServer.uiSettings.unset('doc_table:legacy');
     });
 
-    describe('Customize time range', () => {
+    // FLAKY https://github.com/elastic/kibana/issues/104578
```

Comment:
you're sure this needs to be skipped? the date range you moved to setTimeRange is odd ... fromTime > toTime

Reply:
This describe block was skipped previously, btw, [this PR](https://github.com/elastic/kibana/pull/113597) should unskip it. Actually fromTime < toTime here, since April < August.

Reply:
I agree that we might want to keep it skipped for now, while the underlying problem with flaky time picker selection exists. I added it to the list on https://github.com/elastic/kibana/issues/113520 so we make sure to unskip it, once that problem is fixed.